tasks/register-dcd.yaml

```diff
@@ -1,57 +1,60 @@
 ---
 - name: set facts when discovery IP is set
   set_fact:
-    register_dcd_server_hostname: 'item'
-    register_dcd_server: "{{ hostvars[item]['discoveryip'].split('/')[0] }}"
-    register_dcd_listener: "{{ hostvars[item]['discoveryip'].split('/')[0] }}"
+   register_dcd_server_hostname: 'item'
+   register_dcd_server: "{{ hostvars[item]['discoveryip'].split('/')[0] )}}"
+   register_dcd_listener: "{{ hostvars[item]['discoveryip'].split('/')[0] }}"
   when: hostvars[item].discoveryip is defined
 
 - name: set fact when discovery IP is not set
   set_fact:
-    register_dcd_server_hostname: 'item'
-    register_dcd_server: "{{ hostvars[item].ansible_host }}"
-    register_dcd_listener: "{{ hostvars[item].ansible_host }}"
+   register_dcd_server_hostname: 'item'
+   register_dcd_server: "{{ hostvars[item].ansible_host }}"
+   register_dcd_listener: "{{ hostvars[item].ansible_host }}"
   when: hostvars[item].discoveryip is not defined
 
 - name: Ensure the DCD server IP address is provided
   fail:
-    msg: "Please change the register_dcd_server variable to be the IP address of the DCD device."
+   msg: "Please change the register_dcd_server variable to be the IP address of the DCD device."
   when: register_dcd_server == 'localhost'
 
 - name: Set the listener if it is not specified
   set_fact:
-    register_dcd_listener: "{{ register_dcd_server }}"
+   register_dcd_listener: "{{ register_dcd_server }}"
```

Comment:
I thought so as well, however, yamllint complained that 3 spaces were expected rather than 4. I re-ran integration tests with new indentations and they passed.
example:
  23:5      error    wrong indentation: expected 3 but found 4  (indentation)

I believe this is what is causing the galaxy quality score to be 4.7 rather than 5.0.

Reply:
We could modify the linter if we prefer to conform to our indentation rather then use the defaults: 
https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation

Reply:
ok i see now it looks like it is 2 spaces, Github formatting FTL. if the role executed correctly i have no issue here